docs/game/unreal/getting-started/02-sending-transactions.md

```diff
@@ -9,9 +9,9 @@ This page lists the functionality you can use in your game.
 
 :::tip Write vs. Read methods
 
-* The *Write* method requests — those that change the current state (example: minting NFTs) — incur gas fees to cover smart contract operations. Those requests issue tickets that you need to either approve or reject via MetaMask.
+* The *Write* method requests — those that change the current state (example: minting NFTs) — incur gas fees to cover smart contract operations. Those requests issue tickets you need to approve or reject in MetaMask.
```

Comment:
*Write* requests change the current stage of the blockchain (e.g. a request to mint an NFTs) and thus incur gas fees to cover smart contract operations.